18F/identity-idp
# LG-9865 Remove (In-person) Verify from FSM

## 🎫 Ticket
[LG-9865](https://cm-jira.usa.gov/browse/LG-9865) FSM/Verify - Write delete PR

## 🛠 Summary of changes
- Delete all code related to the step FSM step for Verify and Verify Wait (in person flow)
- Delete all instances of the feature flag `in_person_verify_info_controller_enabled`
- Delete services, views, specs used for the FSM version of Verify for In-Person Proofing

## 📜 Testing Plan
1. Create an account
2. Go to /verify
3. Upload state-issued ID (front + back) to fail ID verification
4. Go to in person proofing
5. Fill in State-Id information, Residential Address, and Social Security Number
6. Poke are around on Verify your information. This is the new non-FSM controller. All visual and functional behavior should be working. Poke around a lot!  Test all buttons, links, and [scenarios](https://www.figma.com/file/t3pPLpCfEyHcVoX7QbTpQV/Milestone-2---Double-Address-Verification?type=design&node-id=606-19639&mode=design&t=fR78qJz55RXho3ZR-0).
7. Proceed in the flow to make a barcode. 

Items to consider testing
- [ ] Team Ada's flow is working as expected
- [ ] Enter a Social Security Number starting with '901' rather than '900' to fail resolution and confirm you are directed back to _/verify/in_person/verify_info_ after you click Try again
- [ ] After clicking Update for State-issued ID, Current residential address, and Social Security Number all go to the correct page and after makes changes you are brought back to _/verify/in_person/verify_info_ 
- [ ] Continue brings you to Verify your phone number
- [ ] Cancel brings you Cancel verifying your identify? (.. and No, keep going brings you right back to _/verify/in_person/verify_info_)
- [ ] Test that you cannot skip head in the flow by changing the url _/verify/in_person/verify_info_ or back once you have passed that step
